bsp/raspberry-pico/board/board.c

```diff
@@ -88,6 +90,15 @@ void rt_hw_board_init()
 #endif
 
 #ifdef RT_USING_CONSOLE
-   rt_console_set_device(RT_CONSOLE_DEVICE_NAME);
+    rt_console_set_device(RT_CONSOLE_DEVICE_NAME);
 #endif
 }
+
+#ifdef RT_USING_MSH
+static int pico_reset(int argc, char *argv[])
+{
+    reset_usb_boot(0, 0);
+    return 0;
+}
+MSH_CMD_EXPORT(pico_reset, Reset Pico to BOOTSEL mode);
```

Comment:
建议用MSH_CMD_EXPORT_ALIAS，可以自定义命令名字
```suggestion
MSH_CMD_EXPORT_ALIAS(pico_reset, reboot, reset Pico to BOOTSEL mode);
```